examples/cycles/test/test.ts

```diff
@@ -18,7 +20,8 @@ const intermediary_canister = createIntermediaryActor(
 );
 
 const tests: Test[] = [
-    ...cleanDeploy('cycles', 'intermediary'), // TODO for now these tests need to be run on a fresh dfx start --clean, since cycles are not discarded on uninstall-code
+    ...deploy('cycles'), // TODO for now these tests need to be run on a fresh dfx start --clean, since cycles are not discarded on uninstall-code
+    ...deploy('intermediary'), // TODO for now these tests need to be run on a fresh dfx start --clean, since cycles are not discarded on uninstall-code
```

Comment:
These explanatory comments might not be necessary now since `deploy` is now the main way to deploy and it's always clean.